dln/operator.py

```diff
@@ -159,7 +161,9 @@ async def _aget_chat_completion_response(self, prompt, **kwargs):
                 )
             except openai.InvalidRequestError as e:
                 # Most likely a content filtering error from Azure.
-                logging.warn(str(e))
+                error_message = f"InvalidRequestError: {e}"
+                logging.warn(error_message)
+                print(colored(error_message, "red"))
                 return str(e)
         else:
```

Comment:
I think the program should crash if it is not due to a ContentFilter error. I'm fine with simply checking if there's any mention of content filtering in the error message. If not, re-raise the exception.